nisystemlink/clients/core/_http_configuration_manager.py

```diff
@@ -78,12 +79,22 @@ def _fallback(cls) -> Optional[core.HttpConfiguration]:
         """
         if cls._configs is None:
             cls._configs = cls._read_configurations()
+
+        if cls._juypter_config is None:
+            try:
+                cls._juypter_config = core.JupyterHttpConfiguration()
+            except KeyError:
+                # Env variables for Jupyter Hub execution are not available.
```

Comment:
Extreme nitpick, but technically this configuration is for notebook execution in general, which isn't always through JupyterHub.

```suggestion
                # Env variables for Jupyter notebook execution are not available.
```